client/verta/tests/unit_tests/strategies.py

```diff
@@ -262,3 +260,98 @@ def mock_workspace(draw):
         )
     )
     return workspace
+
+
+@st.composite
+def mock_pipeline_definition(draw):
+    """Generate a mocked pipeline specification dictionary"""
+
+    # step names in a pipeline must be unique
+    step_names = draw(
+        st.lists(st.text(min_size=5, max_size=25), min_size=5, max_size=5, unique=True)
+    )
+    model_versions = draw(
+        st.lists(
+            st.text(alphabet=["1", "2", "3", "4", "5"], min_size=1),
+            min_size=5,
+            max_size=5,
```

Comment:
```suggestion
            min_size=len(step_names),
            max_size=len(step_names),
```

Reply:
Already done as part of refactoring the fixture